doc/intro.md

```diff
@@ -158,13 +155,13 @@ transaction's values. The second insight of Mimblewimble is that this private
 key can be leveraged to prove ownership of the value.
 
 Alice sends you 3 coins and to obscure that amount, you chose 28 as your
-blinding factor (note that in practice, the blinding factor being a private key, it's an
+blinding factor (note that in practice the blinding factor, being a private key, is an
 extremely large number). Somewhere on the blockchain, the following output appears and
 should only be spendable by you:
 
     X = 28*G + 3*H
```

Comment:
Afain, need to escape markdown syntax with \*

Reply:
Same as above